### PR TITLE
Change prop-types dep to match version of react

### PR DESCRIPTION
**Pull request by nicholascapo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "react-dom": ">15.0.0"
   },
   "dependencies": {
-    "prop-types": "^15.5.10",
+    "prop-types": "^15.0.0",
```
Comment (wesleytodd) [inline package.json:48]: Technically you would want to do `>15.0.0` like it says in the `peerDependencies` if you are trying to get the matching version.  But I am unsure, if this matters, that just moving this into a peer dep is not the better move.  What is this change trying to solve?

Comment (nicholascapo) [reply package.json:48]: One of our services depends on `prop-types@<15.5.0`, and then `npm` complains:

```
npm ERR! peer dep missing: prop-types@>15.5.10, required by @streammedev/flyout@1.0.0
```

I'm just trying to fix the peerDepends error.

Comment (nicholascapo) [reply package.json:48]: But now I'm very confused, and not sure this is a good change at all.

Comment (wesleytodd) [reply package.json:48]: Hm, I think that is a weird error.  But IMO this should probably change to `>15.0.0`, then you could asses if you want to do the work to get them all aligned on version.

File: package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "babel-preset-react": "^6.5.0",
     "mocha": "^3.4.2",
     "npm-check": "^5.4.4",
-    "prop-types": "^15.5.10",
+    "prop-types": "^15.0.0",
```
Comment (hhamilto) [inline package.json:38]: Should we remove this, because we have `prop-types` as a direct dependency already in this package?

Comment (wesleytodd) [reply package.json:38]: If you leave it as a direct dep, and not a peer dep, then for sure it should be :)

**Comment by nicholascapo** [comment]
I fixed this problem in other ways, (and I am convinced that this was the wrong approach). Closing.

**Pull request closed without merging**